### ISSUE TITLE
Cannot log expressions with allowExpressions = true

### ISSUE THREAD

**Issue by matthew1534** [googleapis/cloud-debug-nodejs#769]
# Environment details

  - OS: Docker Container on Google Cloud Run
  - Node.js version: 12.6.0
  - npm version: 6.9.0
  - `@google-cloud/debug-agent` version: 4.0.3
  - the source code isn't in Google Cloud Debugger so I'm manually specifying the file and line in the Google Cloud Debugger Console (`./index.js:22`)

#### Steps to reproduce

I've added `require('@google-cloud/debug-agent').start({ allowExpressions: true });` to the top of the index file however when I try to log 'req' (message: `{req}`) I recieve the following message in the Google Cloud Dugger Console 'Logpoint History' tab:

`logpoint("{req}") Expressions and conditions are not allowed by default. Please set the allowExpressions configuration option to true. See the debug agent documentation at https://goo.gl/ShSm6r.`

As you can see above I have set `allowExpressions: true` and I've tried running multiple requests through the application to no avail. I receive a similar response if I try to add a snapshot.

Thanks very much!

**Comment by matthew1534** [comment]
I'm not sure if this is related or not but on occasion I'm now also seeing this in the logs:
`@google-cloud/debug-agent Failed to re-register debuggee shopify-247011: Error: Could not load the default credentials. Browse to https://cloud.google.com/docs/authentication/getting-started for more information.`

**Comment by soldair** [comment]
thank you so much for the detailed report.
I'm not sure why this is happening yet but i wanted to make sure you know we looked at your issue and we'll follow up with more debug questions

**Comment by matthew1534** [comment]
@soldair thank you, really appreciated - ask away...  :)

I've noticed another issue, I suspect more related to my second comment and I suspect maybe a Cloud Run issue rather than this modules issue but I'll include it as it may help, also, I know how to resolve this message! :

If I haven't assigned the correct permissions to use the debugger to the Google Run service account I receive a log (as I'd expect) `@google-cloud/debug-agent Failed to re-register debuggee project-id: Error: The caller does not have permission` however again these logs are appearing when, theoretically, Cloud Run has no requests. I've been receiving 2 of these logs every minute for the past 4 hours or so. I would expect this message to only show when the service is dealing with a request, maybe I'm missing something. 

**(This is in a different Cloud Run service which is working correctly once the correct permissions are added)**

**Comment by matthew1534** [comment]
In the first service I've now re-added `require("@google-cloud/debug-agent").start({ allowExpressions: true });` to the top of the index.js file and I am no longer receiving any `logpoint("{req}") Expressions and conditions are not allowed by default.` messages (nothing has changed with the service, I've just removed the debugger code & the NPM package and re-added it, so not sure what has gone on here).

In the Google Cloud Debugger Console I can add a logpoint or snapshot and there are no error/warning messages however no logs ever load in the `Logs` tab, just says `Loading logs ...` and then `No older logs match the current filter.` and no log points or snapshots are ever caught.

**Comment by matthew1534** [comment]
**UPDATE:** I've touched nothing and the snapshots are working as expected. The log points are also mostly working as they are logging in Stackdriver Logging however they are not showing up in the Debugger console (none of the logs are showing in the debugger console)

**Comment by DominicKramer** [comment]
Thanks for the feedback and for letting us know about this issue.  I have been able to reproduce what you are seeing where the I can set a logpoint, the logpoint is hit, and the logpoint output appears in the application logs but not in the Logs tab of the Cloud Console Debug UI.

Since the logpoint is hitting and the logpoint output does appear in the application logs, the issue you are seeing looks like an issue with the Debug UI itself and not the Node.js Debug Agent.  As such, I have created an internal issue with the appropriate team to resolve the issue you are seeing.

I have marked your issue as blocked since it needs to be resolved by a different team that is responsible for the Debug UI.  We will keep this thread updated with any updates we receive.  Thanks.
